.rubocop.yml

```diff
@@ -29,21 +23,9 @@ Layout/IndentHash:
 Layout/SpaceInsideHashLiteralBraces:
   Enabled: false
 
-Metrics/ClassLength:
```

Comment:
The teacher_set model is 900 lines long.

Reply:
We can add a one-off exception for that model, and refactor, if appropriate, at some later date.

Reply:
standardize on 600